lib/c.c

```diff
@@ -398,6 +463,7 @@ FILE *fopen(char *filename, char *mode)
 #endif
 
     abort();
+    return -1;
```

Comment:
Is this change necessary?

Reply:
For the present, the function with non-void return type is expected to have a return statement. Maybe we could remove `abort()` and return `NULL`, and then check the `FILE` pointer be valid or not when using?

Reply:
Let's drop the call of `abort`.